.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+<!--
+If your issue is a usage question, submit it to the user mailing-list
+users@riot-os.org or the developer mailing-list devel@riot-os.org.
+-->
+
+#### Description
+<!-- Example: Cannot build gnrc_networking application for samr21-xpro
+board. -->
```

Comment:
Other multli-line comments in your files have the following format.

```md
<!--
Example: Cannot build gnrc_networking application for samr21-xpro board.
-->
```

(I know this doesn't make it a multiline comment anymore, but I think for consistency this is better.

Reply:
(other alternative would be to maybe make it a oneline comment altogether.